pebblo/app/config/config.yaml

```diff
@@ -0,0 +1,8 @@
+daemon:
```

Comment:
I think we are not using this file in the code and I suppose it is kept for reference to users. In that case, I would suggest to keep default values in line with what we have used in the code.

Reply:
Updated the change
resolving.